examples/resources/vantage_team/provider.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
```

Comment:
this isn't necessary for the example, just the resource.tf

Reply:
Can you run `go generate ./...` on your branch as well. Should update some `docs/` with the new resource.